docs/dev_tools/build_profiles.md

```diff
@@ -5,11 +5,15 @@ mbed OS 5 supports three primary build profiles: *develop*, *debug* and *release
 * Small and fast code.
 * Full error information. For example, asserts have file name and line number.
 * Hard to follow code flow when using a debugger.
+* Chip is put to sleep when going idle:
```

Comment:
"The RTOS idle task puts the chip to sleep." is accurate, but maybe too detailed.

"The chip sleeps when idle" is accurate as well.